server/sonar-web/src/main/js/components/SelectList/SelectListListElement.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import * as React from 'react';
+import * as classNames from 'classnames';
+
+export interface SelectListItem {
+  key: string | number;
+  name: string;
+  selected: boolean;
+}
+
+interface Props {
+  element: SelectListItem;
+  active?: boolean;
+  onSelectChange: (element: SelectListItem) => void;
+}
+
+export default class SelectListListElement extends React.PureComponent<Props, {}> {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This is the default value for this type parameter, so it can be omitted. ([typescript:S4157](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS4157))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&pullRequest=3152&issues=AWIkUgx17DpOGbPQ2tAB&open=AWIkUgx17DpOGbPQ2tAB)

[ISSUE_KEY]: AWIkUgx17DpOGbPQ2tAB